src/utils/Config.php

```diff
@@ -441,6 +486,46 @@ public function set($k, $v = true) : void{
 		}
 	}
 
+	public function getString(string $key, string $default = null) : ?string{
+		if(($value = $this->get($key, null)) !== null && !(is_string($value) || is_numeric($value))){
```

Comment:
I am still hesitant about this. `is_numeric` does not mean the string can be losslessly preserved. `1.00` is still converted incorrectly.